product_subscription/models/invoice.py

```diff
@@ -1,50 +1,55 @@
 # -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Houssine Bakkali <houssine@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
 from datetime import datetime
 from openerp import models, fields, api
 
 
 class AccountInvoice(models.Model):
-    _inherit = "account.invoice"
+    _inherit = 'account.invoice'
 
-    subscription = fields.Boolean(string="Subscription")
+    subscription = fields.Boolean(
+        string='Subscription')
     product_subscription_request = fields.One2many(
                 'product.subscription.request',
                 'invoice',
-                string="Product subscription")
+                string='Product subscription')
 
+    @api.multi
     def process_subscription(self, effective_date):
-        # set the subscription request to paid
-        req_vals = {'state': 'paid',
-                    'payment_date': effective_date}
+        self.ensure_one()
 
-        sub_req = self.product_subscription_request
-        sub_template = sub_req.subscription_template
-        # check if there is already an ongoing or an old subscription
-        # tied to the subscriber
+        subscription_template = (
+            self.product_subscription_request.subscription_template
+        )
```

Comment:
that pep8 stuff go to far :)

Reply:
Jme suis détendu par rapport à pep8 mais je préfère comme ça :-)